A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -321,6 +321,42 @@ var A320_Neo_UpperECAM;
                 }
             }
         }
+        getAPWarning(_time) {
```

Comment:
`_` means the variable isn't used.

```suggestion
        getAPWarning(time) {
```